webview/src/plots/components/App.tsx

```diff
@@ -81,6 +82,9 @@ export const feedStore = (
         case PlotsDataKeys.HAS_UNSELECTED_PLOTS:
           dispatch(updateHasUnselectedPlots(!!data.data[key]))
           continue
+        case PlotsDataKeys.PLOT_ERRORS:
```

Comment:
This is a note to self, we should reduce the number of branches in this switch/case by having an enum instead:

```
const actionToDispatch = {
  ...
  [PlotsDataKeys.PLOT_ERRORS]: updatePlotErrors
  [PlotsDataKeys.SELECTED_REVISIONS]: updateSelectedRevisions
  ...
}
```

and a simple `dispatch(actionToDispatch[key](data.data.key))`

Reply:
I'll open an issue for that. And don't worry, it's totally out of scope and has nothing to do with your PR.